math/fityk/Portfile

```diff
@@ -37,19 +37,18 @@ wxWidgets.use       wxWidgets-3.0
 # WARNING: dvipng command 'dvipng' cannot be run (needed for math display), check the pngmath_dvipng setting
 depends_build-append port:swig-lua
 
-boost.version       1.71
 compiler.cxx_standard \
                     2011
 
 use_autoreconf      yes
 
-depends_lib         port:readline \
+depends_lib-append  port:lua \
                     port:ncurses \
-                    port:xylib \
-                    port:lua \
-                    port:zlib \
                     port:python27 \
```

Comment:
can the Python version be updated to v3; it would be ideal to try and get rid of 2.7 where ever possible.

Reply:
I can try with 3.11.